deploy/flux-installer/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for installer.
```

Comment:
Remove all irrelevant config from here, doesn't look like we are using these values